src/less/variables.less

```diff
@@ -78,8 +78,8 @@
 
 @primary-brand-green:         #3CAC30;
 @bright-green:                #3CAC30;
-@body-bg:                     #fff; // @layout6
-@primary-body-bg:             #fff; 
+@body-bg:                     #f8f8f8; // @layout6
+@primary-body-bg:             #fff;
```

Comment:
@pascualin remember the background of the body tag needs to be #f8f8f8. Still seeing #fff as body is pointing to the primary-body-bg variable which has a white value.